.rubocop.yml

```
@@ -19,7 +19,7 @@ Layout/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
 Metrics/LineLength:
-  Max: 200  # Maximum line length from `rails new`
+  Max: 400  # Maximum line length from `rails new`
```

we can add a line annotation to suppress the error rather than change the global configuration.

---

Sounds good, changed.